utils/syntax_highlighting.tsx

```diff
@@ -0,0 +1,184 @@
+hlJS.registerLanguage('go', hljsGo);
+hlJS.registerLanguage('groovy', hljsGroovy);
+hlJS.registerLanguage('handlebars', hljsHandlebars);
+hlJS.registerLanguage('haskell', hljsHaskell);
+hlJS.registerLanguage('haxe', hljsHaxe);
+hlJS.registerLanguage('java', hljsJava);
+hlJS.registerLanguage('javascript', hljsJavascript);
+hlJS.registerLanguage('json', hljsJson);
+hlJS.registerLanguage('julia', hljsJulia);
+hlJS.registerLanguage('kotlin', hljsKotlin);
+hlJS.registerLanguage('less', hljsLess);
+hlJS.registerLanguage('lisp', hljsLisp);
+hlJS.registerLanguage('lua', hljsLua);
+hlJS.registerLanguage('makefile', hljsMakefile);
+hlJS.registerLanguage('markdown', hljsMarkdown);
+hlJS.registerLanguage('matlab', hljsMatlab);
+hlJS.registerLanguage('objectivec', hljsObjectivec);
+hlJS.registerLanguage('ocaml', hljsOcaml);
+hlJS.registerLanguage('perl', hljsPerl);
+hlJS.registerLanguage('php', hljsPhp);
+hlJS.registerLanguage('powershell', hljsPowershell);
+hlJS.registerLanguage('puppet', hljsPuppet);
+hlJS.registerLanguage('python', hljsPython);
+hlJS.registerLanguage('r', hljsR);
+hlJS.registerLanguage('ruby', hljsRuby);
+hlJS.registerLanguage('rust', hljsRust);
+hlJS.registerLanguage('scala', hljsScala);
+hlJS.registerLanguage('scheme', hljsScheme);
+hlJS.registerLanguage('scss', hljsScss);
+hlJS.registerLanguage('smalltalk', hljsSmalltalk);
+hlJS.registerLanguage('sql', hljsSql);
+hlJS.registerLanguage('stylus', hljsStylus);
+hlJS.registerLanguage('swift', hljsSwift);
+hlJS.registerLanguage('tex', hljsTex);
+hlJS.registerLanguage('vbnet', hljsVbnet);
+hlJS.registerLanguage('vbscript', hljsVbscript);
+hlJS.registerLanguage('verilog', hljsVerilog);
+hlJS.registerLanguage('vhdl', hljsVhdl);
+hlJS.registerLanguage('xml', hljsXml);
+hlJS.registerLanguage('yaml', hljsYaml);
+
+type languageObject = {
+    [key: string]: { 
+        name: string,
+        extensions: string[],
+        aliases?: string[]
+    }
+}
+
+const HighlightedLanguages: languageObject = Constants.HighlightedLanguages;
+
+export function highlight(lang: string, code: string): string {
+    const language = getLanguageFromNameOrAlias(lang);
+
+    if (language) {
+        try {
+            return hlJS.highlight(language, code).value;
+        } catch (e) {
+            // fall through if highlighting fails and handle below
+        }
+    }
+
+    return TextFormatting.sanitizeHtml(code);
+}
+
+export function getLanguageFromFileExtension(extension: string): string | null {
+    for (var key in HighlightedLanguages) {
+        if (HighlightedLanguages[key].extensions.find((x: string) => x === extension)) {
+            return key;
+        }
+    }
+
+    return null;
+}
+
+export function canHighlight(language: string): boolean {
+    return Boolean(getLanguageFromNameOrAlias(language));
+}
+
+export function getLanguageName(language: string): string {
+    if (canHighlight(language)) {
+        return HighlightedLanguages[getLanguageFromNameOrAlias(language)].name;
```

Comment:
Hey @FlaviaBastos, you should be able to do an explicit null check before this line, which should clear up the TS error. Something like: 
```suggestion
        const name: string | null = getLanguageFromNameOrAlias(language);
        if (!name) return '';
        return HighlightedLanguages[getLanguageFromNameOrAlias(language)].name;
```

Reply:
Great! Thanks for the suggestion @devinbinnie. 
I just pushed an update. Please let me know how it looks like.